packages/mattermost-redux/src/actions/users.ts

```diff
@@ -66,74 +66,6 @@ export function createUser(user: UserProfile, token: string, inviteId: string, r
     };
 }
 
-export function login(loginId: string, password: string, mfaToken = '', ldapOnly = false): ActionFunc {
```

Comment:
the naming is not correct since it means login but actually we are also calling load me etc in this function. turns out we dont need this is one action anyways. Also the login action needs to be separate as it never errors out on mfa enforce settings.